Automattic/woocommerce-payments
# Exclude `Estimated` deposits from query when downloading deposits as a CSV

Fixes #7631

Based on PR #7605

> [!note]
> This implementation is not currently working due to server-side concatenation of estimated deposits – the `status_is_not` filter is not sufficient.
> This will need to be deployed alongside a server-side change. See https://github.com/Automattic/woocommerce-payments-server/issues/4146.

#### Changes proposed in this Pull Request









#### Testing instructions





* Preferably, test using a store with 25+ deposits including 1+ estimated deposits.
	* Estimated deposits won't show up in the UI due to #7605, but they will exist on the server.
* View **Payments → Deposits** and set the rows per page to 25.
* Press the **Download** button and the **CSV** will be emailed to you. (Check WP Mail Log if using local server).
* On `develop`, the CSV will include estimated deposits.
* On this branch, ensure the CSV doesn't include estimated deposits.

-------------------


**Post merge**
